### DISCUSSION TITLE
Distributions and curated sets of el-get packages

### DISCUSSION THREAD

**Discussion by elliottslaughter** [dimitri/el-get#2870] [Ideas]
As has been noted in https://github.com/dimitri/el-get/issues/2869, fresh el-get installations often fail because the ecosystem of el-get packages is a bit of a wild west. It was pointed out in a comment that el-get itself is not necessarily intended to be a place for the curation of stable packages; but this is potentially something that could be added as a first-class construct in some form https://github.com/dimitri/el-get/issues/2869#issuecomment-1219402682.

I think there are two questions for this discussion to address:

 1. What should the shape of this feature be? I think Quicklisp is a nice inspiration here, because if I'm not mistaken, it provides the ability to maintain distributed distributions (i.e., if you don't like Quicklisp's distribution you can create your own): https://www.quicklisp.org/beta/.
 2. Who has time and inclination to work on it?

I'll just mention up front that unfortunately I cannot commit any substantial time to this. I can provide whatever guidance I can, and perhaps testing (once there is something to test), but unfortunately I'm not going to be able to offer much in the way of implementation effort.

**Comment by lispstudent** [comment]
As I mentioned earlier,  I am not a a proficient elisp-coder, especially for el-get standard, but count on me for testing. I am on FreeBSD.
